functional-tests/lib/app.js

```diff
@@ -8,6 +8,7 @@ const bodyParser = require('body-parser');
 const partials = require('express-partial-templates');
 const template = require('hof-govuk-template');
 const cookieParser = require('cookie-parser');
+const mockPostcode = require('../../mock-postcode');
```

Comment:
thanks @lennym I've updated with now the explicit port number.  Hopefully, you are OK with where I put the config.  I had it outside but since it's related to tests, I put it in the functional-test folder